src/api/app/views/webui2/webui/request/_maintainer_link.html.haml

```diff
@@ -0,0 +1,6 @@
+- if maintainers.size <= 3
+  = 'Package maintainer'.pluralize(maintainers.size) + ':'
+  - maintainers.pluck(:login).each do |maintainer|
+    = user_with_realname_and_icon(maintainer, no_icon: true, short: true) + (',' unless maintainer.equal? maintainers.last[:login])
```

Comment:
Well done!

Just some things:

- please add the parenthesis on  ```maintainer.equal?(maintainers.last[:login])```
- can you somehow come up with a solution similar to ```Array#to_sentence``` to the logins?  (i.e user_1, user_2 and user_3)?
- Nitpick: maybe too much logic to the view? (this could be extracted to a helper method)

Reply:
Good feedback, thank you!

I think the helper method is much cleaner approach.